partcad/src/partcad/__init__.py

```
@@ -57,18 +57,44 @@
+# TODO(clairbee): move the below to `logging_sentry.py`
 if not sentry_sdk.is_initialized() and user_config.get_string("sentry.dsn"):
+    critical_to_ignore = [
+        "action_start: ",
+        "action_end: ",
+    ]
+    debug_to_ignore = [
+        "Starting action",
+        "Finished action",
+    ]
+
+    def before_send(event, hint):
```

Ruff (0.8.2)

Missing return type annotation for private function `before_send`

(ANN202)